src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -288,7 +351,8 @@ def initialize_processor_from_path(
 
 def main(
     model_args: ModelArguments,
-    data_args: DataTrainingArguments,
+    data_args: DatasetArguments,
+    recipe_args: RecipeArguments,
```

Comment:
Why does oneshot live in the training arguments?
I dont think we want it coming from main at all